ITEvents/Screens/Event view/EventView.swift

```diff
@@ -19,16 +19,12 @@ class EventView: UIStackView {
                                                          address: event.address)
         eventView.addArrangedSubview(eventPlace)
         
-        let eventDescription = EventDescriptionView.initiateAndSetup(with: tempDescription)
-        //TODO event.description)
+        let eventDescription = EventDescriptionView.initiateAndSetup(with: event.description)
         eventView.addArrangedSubview(eventDescription)
         
         let eventTags = EventTagsView.initiateAndSetup(with: event.tags)
         eventView.addArrangedSubview(eventTags)
         
-        let showEventSource = ShowEventSourceView.initiateAndSetup(with: "Show on Timapad")
```

Comment:
why this is removed? looks like logically it should be here

Reply:
if let source = event.source, let url = event.url {
let showEventSource = ShowEventSourceView.initiateAndSetup(with: source, and: url)
//TODO set showEventSource.souceOpenAction = ...